### PR TITLE
Percent brackets

### PR DESCRIPTION
**Pull request by haaavk**
Show session percents in operating systems, browsers and device types.

### CODE REVIEW

File: shynet/dashboard/templatetags/helpers.py [hunk 1]
```diff
@@ -184,3 +184,9 @@ def urldisplay(url):
         )
     else:
         return url
+
+
+@register.filter
+def percent(value, total):
+    percent = value / total
```
Comment (milesmcc) [inline shynet/dashboard/templatetags/helpers.py:191]: Could we just add a quick check if `total` is `0`? So:

```suggestion
    if total == 0:
        return "N/A"
    percent = value / total
```

**Comment by milesmcc** [comment]
Awesome, thanks! Just take a look at that one comment and then we'll be set to merge into the next release.

**Comment by CasperVerswijvelt** [comment]
Also show percentage for referrer, country and operating system?

**Comment by haaavk** [comment]
Division by zero fixed. 
Operating system was always implemented.
Country will be replaced with map possibly?
I'm not sure if referrer should have percents. 
If everybody wants percents in all data I can add it.

**Comment by CasperVerswijvelt** [comment]
My bad, I just meant the other cards. I think it would be best to show it consistently in all cards.

About country: my original tought was to just have the data available in both map and table form, feel free to comment on that issue #92 where I posted some screenshots. Still don't know what form it should be and where it should be placed on the dashboard.

**Comment by haaavk** [comment]
I added percent to country card. 
I have problems with referral card size on my small laptop even without percents. 
Long links trigger horizontal scroll.
Shynet - referral - Screenshot 2021-04-30 09:01:26

**Comment by CasperVerswijvelt** [comment]
That horizontal scroll issue is something I fixed in #120

**Comment by haaavk** [comment]
What do you think about visualization of percents? For example with progress bar.
Shynet - progress Screenshot 2021-05-06 16:43:42

**Comment by CasperVerswijvelt** [comment]
I like it, but I would make the blue a bit lighter so it looks subtler. Also maybe extend the visualization to the full width of the table

**Comment by milesmcc** [comment]
I agree re: extending it to be the full width of the table. This looks great! And I'm glad you are using the built-in color theme. You might consider making it lighter by setting the opacity (e.g., `opacity-75` or `opacity-50`).

**Comment by milesmcc** [comment]
Hey @haaavk! Just wanted to share a few updates/requests for this PR. First, thank you for working on it! I just merged all your other PRs, and am excited to merge this one ASAP as well.

Two quick things:

1. @CasperVerswijvelt made several UX/style improvements, mostly to do with flex arrangements and layouts on the service page. Would you be able to rebase/rework in order to keep those improvements?
2. Right now, putting the percentages on their own line causes everything to be a bit too spaced out. I wonder if you could put them to the right of the raw number, not unlike what you originally did.

Finally, I wonder if we could maybe make this look even better by putting a low-opacity progress bar behind the text itself, sort of like what Plausible does:

image

That might help us maintain high information density while also adding this visual nicety.

**Pull request closed without merging**